docs/reference/files.md

```diff
@@ -597,22 +601,68 @@ Not supported by GraphQL
 </template>
 <template #sdk>
 
+#### Web
+
+::: code-group
+
+```js-vue [index.js]
+import { createDirectus, rest, uploadFiles } from 'https://unpkg.com/@directus/sdk@{{ packages['@directus/sdk'].version.major }}';
+
+const client = createDirectus('https://directus.example.com').with(rest());
+
+const form = document.querySelector('#upload-file');
+
+if (form && form instanceof HTMLFormElement) {
```

Comment:
```suggestion
const form = document.getElementById('upload-file');

if (form) {
```
Considering you have the HTML defined right there i think the HTMLFormElement check may be a little overkill.

Reply:
Copied that from an example for the older SDK, but I agree that this is a tad overkill 😄👍